### PR TITLE
Use translations for Ledger Manager instead of hardcoded value

### PR DESCRIPTION
**Pull request by KVNLS**
### 📝 Description

Allow translation for Ledger Manager. Impacts 'Allow Ledger Manager' screen and 'Rename my Ledger'

### ❓ Context

- **Impacted projects**: `llm` `lld` 
- **Linked resource(s)**: `https://ledgerhq.atlassian.net/browse/LIVE-8915` 

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

### 📸 Demo

image
image
image

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/ledger-live-desktop/src/renderer/components/DeviceAction/rendering.tsx [hunk 1]
```diff
@@ -406,11 +406,11 @@ export const renderListingApps = () => (
 export const renderAllowManager = ({
   modelId,
   type,
-  wording,
+  isRenameRequest,
 }: {
   modelId: DeviceModelId;
   type: Theme["theme"];
-  wording: string;
+  isRenameRequest?: boolean;
```
Comment (alexandremgo) [inline apps/ledger-live-desktop/src/renderer/components/DeviceAction/rendering.tsx:413]: Not mandatory at all, but maybe a union of string literals (enum) could be more flexible ? Like `requestType: "renaming" | "manager"` with a default to `"manager"` ? WDYT ?

**Review by alexandremgo — APPROVED** [review]
Looks good ! Thank you :)

File: apps/ledger-live-desktop/src/renderer/components/DeviceAction/rendering.tsx (hunk 1 above)
Comment (KVNLS) [reply apps/ledger-live-desktop/src/renderer/components/DeviceAction/rendering.tsx:413]: I'm good with your proposal, it's more open to changes. I will do it, Thanks

**Review by mcayuelas-ledger — APPROVED** [review]
LGTM

**Pull request merged**